log_outgoing_requests/utils.py

```diff
@@ -0,0 +1,155 @@
+import logging
+from typing import TYPE_CHECKING, Iterable, Optional, Union
+
+from django.conf import settings
+
+from requests import Request, Response
+
+if TYPE_CHECKING:  # pragma: nocover
+    from .models import OutgoingRequestsLogConfig
```

Comment:
is there an actual circular import here? I don't see the `utils` module imported in `models.py`